gravitee-apim-console-webui/src/management/api/component/plan/api-plan-form.component.ts

```diff
@@ -139,6 +139,8 @@ export class ApiPlanFormComponent implements OnInit, AfterViewInit, OnDestroy, C
   @Input()
   planStatus?: PlanStatus;
 
+  @Input() isTcpApi!: boolean;
```

Comment:
```suggestion
  @Input({ required: true }) isTcpApi!: boolean;
```
if you use the `!` It makes more sense to add it
Hint new for angular 16 +.